### DISCUSSION TITLE
using matrix as a chat service for mylyn

### DISCUSSION THREAD

**Discussion by BeckerFrank** [eclipse-mylyn/.github#22] [Ideas]
Do we want to use [eclipse chat service](https://chat.eclipse.org/#/room/#eclipse-projects:matrix.eclipse.org)?

If yes what rooms should we use?

possible options.

1. one space for mylyn and mylyn doc
2. two separate spaces for mylyn and mylyn doc
3. other layout, maybe more then one space (general, context, task connector, build connector, docs,  ...) 

Thougths?

**Comment by akurtakov** [comment]
I feel overloaded so I'll have to restrict myself to github/mailing list only for Mylyn.

**Comment by ruspl-afed** [comment]
both option 1 and option 2 are ok with me, option 3 looks not viable

perhaps we need to start from option 1 and see how active this chat could be

**Comment by BeckerFrank** [comment]
Issue https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3013 for option 1

**Comment by BeckerFrank** [comment]
The room is available under https://chat.eclipse.org/#/room/%23tools.mylyn:matrix.eclipse.org
